cypress/integration/project-error-screen.spec.ts

```diff
@@ -0,0 +1,46 @@
+import capitalize from "lodash/capitalize";
+import mockProjects from "../fixtures/projects.json";
+
+describe("Project Error Screen", () => {
+  it("should checkout that error message exists when there is an error", () => {
+    const languageNames = ["React", "Node.js", "Python"];
+    const statusText = ["critical", "warning", "stable"];
+
+    cy.intercept("GET", "https://prolog-api.profy.dev/project", (req) =>
+      req.destroy()
+    ).as("getProjectsFailed");
+
+    // open projects page
+    cy.visit("http://localhost:3000/dashboard");
+
+    cy.wait(6000);
+    cy.wait("@getProjectsFailed");
+
+    cy.get("[data-cy='project-error-message']")
+      .contains("There was a problem whle loading the project data")
+      .should("exist");
+
+    // waits 2 seconds to be able to see the component render
+    cy.wait(2000);
```

Comment:
Is this necessary? Usually waiting for seemingly arbitrary periods is discouraged (if not required). Cypress anyways waits for some time (I think the timeout is 5s) before the test fails due to a missing element. I'd try to remove this line and see if it still fails (if you haven't done so already)